tripy/examples/segment-anything-model-v2/image_demo.py

```diff
@@ -174,19 +174,22 @@ def main(image_path: str, save_path: Optional[str] = None):
     # Create predictor and process image
     predictor = SAM2ImagePredictor(sam2_model)
 
-    predictor.set_image(image)
+    predictor.set_image_batch([image, image])
```

Comment:
I think we should an option to run batch N inference and not always batch = 2

Reply:
Added `--batch` option. The `set_image_batch` API expects a list of `Image` objs for now, it will do transformations and pack them into a batched tensor.